ddtrace/contrib/logging/patch.py

```diff
@@ -59,6 +70,27 @@ def _w_makeRecord(func, instance, args, kwargs):
     return record
 
 
+def _w_StrFormatStyle_format(func, instance, args, kwargs):
+
+    # The format string "dd.service={dd.service}" expects
+    # the record to have a "dd" property which is an object that
+    # has a "service" property
+    # PercentStyle, and StringTemplateStyle both look for
+    # a "dd.service" property on the record
+    record = kwargs.get("record", args[0])
+
+    dd = DDLogRecord(
+        trace_id=getattr(record, RECORD_ATTR_TRACE_ID, RECORD_ATTR_VALUE_ZERO),
+        span_id=getattr(record, RECORD_ATTR_SPAN_ID, RECORD_ATTR_VALUE_ZERO),
+        service=getattr(record, RECORD_ATTR_SERVICE, ""),
+        version=getattr(record, RECORD_ATTR_VERSION, ""),
+        env=getattr(record, RECORD_ATTR_ENV, ""),
+    )
+    setattr(record, "dd", dd)
```

Comment:
I take it this means that `record.dd = dd` doesn't work for some reason?

Reply:
now you have me wondering what case I hit that meant I needed this....

this is what we do above in `_w_makeRecord` as well... but it seems like `record.dd = dd` should work 🤔

Reply:
oh that is right, I needed it above because we are using `dd.<thing>` as the attribute name.

Good catch I'll update this to be `record.dd = dd`